<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UrlShortenerWidget</class>
 <widget class="QWidget" name="UrlShortenerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="chbAuto">
     <property name="text">
      <string>&amp;Automatically shorten links before posting</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblShortcut">
     <property name="text">
      <string>&amp;Shortcut:</string>
     </property>
     <property name="buddy">
      <cstring>lineShortcut</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineShortcut">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboShorten"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblshorten">
     <property name="text">
      <string>Shorten &amp;links via</string>
     </property>
     <property name="buddy">
      <cstring>comboShorten</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
